src/app/asset-page/asset-page.component.ts

```diff
@@ -782,23 +785,25 @@ export class AssetPage implements OnInit, OnDestroy {
         .subscribe((blob) => {
           if (blob.size > 0) {
               if (this.isMSAgent) {
-                  this.downloadLoading = false
-                  this.navigator.msSaveBlob(blob, 'download.jpg')
+                  // We give MS Browsers an extra pause of 2 secs
```

Comment:
could you add the reason for why we have the pause? It's apparent from the code that there is a pause, so adding reasoning here would help explain the situation to future code readers

Reply:
The extra pause in Windows browsers is there to avoid the file save dialog from triggering before rundownLoadView fully returns